site2/website/blog/2021-02-03-Five-Minute-Interview-Elastic.md

```diff
@@ -0,0 +1,45 @@
+---
+author: Jonathan Ellis
+authorURL: https://twitter.com/spyced
+title: Five Minute Interview: Apache Pulsar with Elastic
+---
+
+This is the first in a series of quick interviews with members of the Apache Pulsar community to learn more about how Pulsar is being used in the wild.  Our guest today is [Ricardo Ferreira](https://twitter.com/riferrei).
+
+<!--truncate-->
+
+### What industry do you work in and what is your role?
+
+I work in the Technology/SaaS space. I work for Elastic, the company behind the Elastic Stack and Elastic Cloud. There, I am part of the community team focused on North America, working as a developer advocate.
+
+### What projects / use cases led to you adopting Pulsar?
```

Comment:
Would it better if we use 
"What projects / use cases led you to adopt Pulsar?"
or 
"What impacted your decision to adopt Pulsar?"

Reply:
It didn't work out that way in this specific case, but in the future I'm trying to get people to talk about, "I'm using Pulsar to solve problem X."  So I'd prefer to leave this the way it is.

Reply:
That's ok.